examples/Google.Ads.GoogleAds.Examples.csproj

```diff
@@ -11,6 +11,8 @@
     <IncludeSource>true</IncludeSource>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <SignAssembly>true</SignAssembly>
```

Comment:
Is this signing new or was it removed before on mistake?  I'm wondering why we're adding this now for these examples.

Reply:
Yes, this signing is new. This is needed because 
- We have internal fields in client library that's exposed to test assembly only through InternalsVisibleTo annotation.
- Using InternalsVisibleTo requires signing of the Test assembly.
- Once the test assembly is signed, all its dependencies would have to be signed.
- I added some integration tests for examples, which makes examples project a dependency of the test project. So examples assembly have to be signed.